source/developer/slash-commands.rst

```diff
@@ -171,9 +171,9 @@ Tips and Best Practices
 Share Your Integration
 ----------------------
 
-If you've built an integration for Mattermost, please consider `sharing your work <https://mattermost.org/share-your-mattermost-projects>`__ in our `app directory <https://about.mattermost.com/default-app-directory>`__.
+If you've built an integration for Mattermost, please consider `sharing your work <https://developers.mattermost.com/integrate/getting-started/>`__ in our `app directory <https://mattermost.com/marketplace/#privateApps>`__.
 
-The `app directory <https://about.mattermost.com/default-app-directory>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or self-managed infrastructure.
+The `app directory <https://mattermost.com/marketplace/#privateApps>`__ lists open source integrations developed by the Mattermost community and are available for download, customization and deployment to your private cloud or self-managed infrastructure.
```

Comment:
Curious as to the inclusion of "#privateApps" in the URL?

Reply:
Ahh it was coming in the URL...so I put the URL without changing anything

Reply:
Looks like there's an auto redirect from ``https://about.mattermost.com/default-app-directory`` to ``https://mattermost.com/marketplace/#privateApps``. Good catch, @Sayanta66!

Reply:
Thanks a lot @cwarnermm